vllm/model_executor/models/utils.py

```diff
@@ -748,3 +749,59 @@ def fast_topk(values, topk, dim):
     else:
         # Use topk for efficiency with larger k values
         return torch.topk(values, topk, dim=dim)
+
+
+def should_use_multi_stream() -> bool:
+    """Utility function to check if multi-stream should be enabled.
+    Multi-stream is only enabled when cuda graph is turned on because switch
+    stream has extra host overhead.
+    """
+    return current_platform.is_cuda() \
+        and torch.cuda.is_current_stream_capturing()
```

Comment:
known issue: This line is breaking torch.compile(). I need another way to tell if we are currently in cuda graph capturing mode...